docs/reference/run.md

```diff
@@ -189,6 +189,82 @@ round-trip min/avg/max = 0.257/0.288/0.326 ms
 For more information about container networking, see [Networking
 overview](https://docs.docker.com/network/)
 
+## Filesystem mounts
+
+By default, the data in a container is stored in an ephemeral, writable
+container layer. Removing the container also removes its data. If you want to
+use persistent data with containers, you can use filesystem mounts to store the
+data persistently on the host system. Filesystem mounts can also let you share
+data between containers and the host.
+
+Docker supports two main categories of mounts:
+
+- Volume mounts
+- Bind mounts
```

Comment:
Related to my previous comment. Looks like this section is not as complete as https://docs.docker.com/storage/, and misses at least the `tmpfs` mounts.

Reply:
yeah I didn't intend to make everything on this page exhaustive; it could put us back where this page started. My thinking is:

- General information here
- All the details in the reference docs